pom.xml

```diff
@@ -452,6 +452,11 @@
                 <artifactId>mongo-spark-connector_${scala.binary.version}</artifactId>
                 <version>2.4.1</version>
             </dependency>
+            <dependency>
+                <groupId>net.snowflake</groupId>
+                <artifactId>spark-snowflake_${scala.binary.version}</artifactId>
+                <version>2.16.0-spark_3.3</version>
```

Comment:
The connector version `2.16.0-spark_3.3` specifically says it's compiled for the Spark 3.3. What about the other Spark versions? Will the plugin still be binary compatible?